test/unit/util/tile_cover.test.js

```diff
@@ -1,95 +0,0 @@
-import {test} from '../../util/test';
-import tileCover from '../../../src/util/tile_cover';
-import MercatorCoordinate from '../../../src/geo/mercator_coordinate';
-import {OverscaledTileID} from '../../../src/source/tile_id';
-
-test('tileCover', (t) => {
-
-    t.test('.cover', (t) => {
-        t.test('calculates tile coverage at w = 0', (t) => {
-            const z = 2,
-                coords = [
-                    new MercatorCoordinate(0, 0.25),
-                    new MercatorCoordinate(0.25, 0.25),
-                    new MercatorCoordinate(0.25, 0.5),
-                    new MercatorCoordinate(0, 0.5)
-                ],
-                res = tileCover(z, coords, z);
-            t.deepEqual(res, [new OverscaledTileID(2, 0, 2, 0, 1)]);
-            t.end();
-        });
-
-        t.test('calculates tile coverage at w > 0', (t) => {
-            const z = 2,
-                coords = [
```

Comment:
Could we repurpose the same tests to test Transform.coveringTiles (seems like a square viewport and no pitch nor heading)? Or, probably better, have other tests, covering different value of wrap (< 0, > 0...), coverage across meridians?

Reply:
This is a very good point. I've included new tests that should cover these cases.